package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "tape test/*.test.js",
     "dev": "live-server --port=8000",
-    "dev:alt": "serve -l 8000",
+    "serve": "serve -l 8000",
     "test:dev": "tape ./redesign/test/*.test.js",
```

Comment:
I can't see the `redesign` folder in the proejct, so the `test:dev` command won't run any tests either